GVFS/GVFS.SignFiles/GVFS.SignFiles.csproj

```diff
@@ -36,7 +36,6 @@
   <ItemGroup>
     <FilesToSign Include="
       $(BuildOutputDir)\GitHooksLoader\bin\$(Platform)\$(Configuration)\GitHooksLoader.exe;
-      $(BuildOutputDir)\FastFetch\bin\$(Platform)\$(Configuration)\FastFetch.exe;

```

Comment:
We don't have a FastFetch.exe at this point as it's created by `dotnet publish`. We could do something like what Ameen's installer PR does to run a publish as part of a build, but I'm not sure how the signing infrastructure works and if it would be able to handle a .NET Core binary. Additionally, now that FastFetch is .NET Core, the SignFiles .NET Framework project can't take a build dependency on it anymore. I would hope we could make SignFiles .NET Standard to work around this but I'll wait for @jrbriggs to get back.

This is something else I want to follow up with.

Reply:
If we don't have a work item already to sign FastFetch.exe, can you create one?

Reply:
If we don't have a work item already to sign FastFetch.exe, can you create one?

Reply:
We have #574 for figuring out the future of distributing FastFetch. I assume that would cover the work to sign it. Would you prefer that I open a separate issue?

Reply:
> We have #574 for figuring out the future of distributing FastFetch. I assume that would cover the work to sign it. Would you prefer that I open a separate issue?

Could you file a separate issue?  The signing issue is separate from how the binaries are distributed.

/ cc: @jrbriggs

Reply:
#623